README.md

```diff
@@ -1,7 +1,16 @@
 # AwsCommonRuntimeKit
-The AWS CRT for Swift is currently in developer preview and is intended strictly for feedback purposes only. 
+The AWS CRT for Swift is currently in developer preview and is intended strictly for feedback purposes only.
 Do not use this for production workloads.
 
+# AWS CRT Swift
+Swift bindings for the AWS Common Runtime.
```

Comment:
Maybe we should consolidate to one? I left the top line title alone because it seemed important to notify users explicitly and separately that this is in Dev Preview. Assumed we could remove it once it goes GA.

Reply:
Yeah, we should consolidate to one as two titles are confusing.